torch/csrc/jit/export.cpp

```diff
@@ -497,10 +503,23 @@ GraphEncoder::GraphEncoder(
 
   EncodeGraph(model_proto_.mutable_graph(), graph, initializers, dynamic_axes, keep_initializers_as_inputs);
 
-  for (const std::string& domain : domains_) {
+  for (auto domain : domains_) {
     auto* opset = model_proto_.add_opset_import();
     opset->set_domain(domain);
-    opset->set_version(0);
```

Comment:
If this change is a breaking change for the caffe2/aten domain, another option would be to set_version to 0 for caffe2/aten domain, and otherwise, follow the new logic.
This way you wouldn't have to change the tests in test_operators.py.

Reply:
Yes, this would need a workaround. I'm interested to see if this change can go through without breaking any Caffe2 tests. If there are failures, I'll add that.

Reply:
First, it's a breaking change. Good catch @lara-hdr. But it is probably fine for caffe2 users, since most of them don't pay attention to the opset version. Only opset domain matters. So I think we can land this as it it.

Reply:
Thanks